pkgs/applications/misc/hyper/default.nix

```diff
@@ -11,11 +11,11 @@ let
   ];
 in
 stdenv.mkDerivation rec {
-  version = "0.7.6";
-  name = "hyperterm-${version}";
+  version = "0.8.3";
+  name = "hyper-${version}";
   src = fetchurl {
```

Comment:
Better to use `fetchFromGitHub { owner = "zeit"; repo = "hyper"; rev = "v${version}"; sha256 = "..." }`.